vcd/resource_vcd_vapp_network.go

```diff
@@ -515,7 +515,7 @@ func resourceVcdVappNetworkImport(d *schema.ResourceData, meta interface{}) ([]*
 	}
 
 	if vappNetworkToImport.Configuration.FenceMode == types.FenceModeBridged ||
-		(vappNetworkToImport.Configuration.FenceMode == types.FenceModeNAT && vappNetworkToImport.Configuration.Features.DhcpService == nil) {
+		(vappNetworkToImport.Configuration.FenceMode == types.FenceModeNAT && vappNetworkToImport.Configuration.IPScopes.IPScope[0].IsInherited == true) {
```

Comment:
We should check for `vappNetworkToImport.Configuration.IPScopes` to be not nil , and for `vappNetworkToImport.Configuration.IPScopes.IPScope` to have elements

Reply:
it always return, but added